src/components/renders/bootstrap/edit/edit-render.tsx

```diff
@@ -69,6 +69,35 @@ export class EditRender implements Renderer {
     event.stopPropagation();
   }
 
+  setReadonlyContent(component, initialContent) {
```

Comment:
More than setting, I believe that with this method we are getting. What do you think?

Reply:
That method takes the content and returns the content with a new format when the component is readonly mode and it has a specific type. 🤔
What name do you suggest?

Reply:
If it returns something, `getReadonlyContent` sounds more accurate.

Reply:
Renamed! Look in 0be53e5